kalliope/signals/order/order.py

```diff
@@ -53,6 +53,11 @@ def __init__(self):
         self.trigger_callback_called = False
         self.is_trigger_muted = False
 
+        # If kalliope is asked to start muted
+        #self.set_mute_status(self.settings.start_muted)
+        if self.settings.start_options['muted'] is True:
```

Comment:
are you sure that the state machine will not unmute it anyway?

Reply:
Well, during my tests, it did work and kalliope was rightly muted ^^
Do you want debug log to see results or smth else?

Reply:
No, I believe you ;).

I'll test this one this weekend before merging it. Thanks for your work !